modules/upnp/doc_classes/UPNP.xml

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="UPNP" inherits="RefCounted" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
 	<brief_description>
-		UPNP network functions.
+		Universal Plug and Play (UPnP) functions for network device discovery, querying and port forwarding.
 	</brief_description>
 	<description>
-		Provides UPNP functionality to discover [UPNPDevice]s on the local network and execute commands on them, like managing port mappings (port forwarding) and querying the local and remote network IP address. Note that methods on this class are synchronous and block the calling thread.
-		To forward a specific port:
+		This class can be used to discover compatible [UPNPDevice]s on the local network and execute commands on them, like managing port mappings (for port forwarding/NAT traversal) and querying the local and remote network IP address. Note that methods on this class are synchronous and block the calling thread.
+		To forward a specific port (here [code]7777[/code], note both [code]discover()[/code] and [code]add_port_mapping()[/code] can return errors that should be checked):
```

Comment:
Didn't you literally just update the docs that teach you about this? :)
```suggestion
		To forward a specific port (here [code]7777[/code], note both [method discover] and [method add_port_mapping] can return errors that should be checked):
```

Reply:
Yup, and because of that I thought of `[param]` too, and then I wondered whether that was backported, found out its not and then decided to leave this as is so it can be easily cherry-picked... and prompty forgot about the `[method]` switch.

Oof. :P